portfolio/src/main/webapp/style.css

```diff
@@ -73,36 +112,67 @@ section.blue {
 }
 
 .profile {
-  max-width: 1000px;
+  max-width: 1300px;
   text-align: left;
+  margin-bottom: 100px;
 }
 
 .projects {
   max-width: 1500px;
-}
-
-.content {
-  text-align: left 
+  display: flex; 
+  flex-flow: row wrap;
 }
 
 .intro {
-  margin: 0;
-  padding-left: 10px;
+  margin: auto;
+  padding-left: 20px;
 }
 
 #fact-container {
-  height: 20px;
-  margin-top: 20px;
+    height: 20px;
+    margin-top: 20px;
+    color: inherit;
+    text-align: center;
+}
+
+.to-do-container {
+  /* text-align: left */
+  margin: auto;
+}
+
+.to-do-container p {
+  margin: 30px;
+}
+
+.to-do-container h4 {
+    font-size: 20px;
+    margin-bottom: 45px;
 }
 
 /*Cards*/
+.profile-card {
+  border-radius: 8px;
+  overflow: hidden;
```

Comment:
What is this for?

Reply:
for the 3 columns under the about me paragraph

Reply:
Oh ok I don't think this actually does anything?

Reply:
sorry, are you referring to the overflow: hidden line?

Reply:
Yeah!

Reply:
I thought it cut text off if the screen got too small for the size of the card?  I was thinking maybe i should change it to overflow: scroll?  But perhaps that situation shouldn't ever happen? (I will need to improve the responsive to page size @media things, bc it doesn't look good on a phone.)

Reply:
Since there's no height set on the element, when the width shrinks the height will get taller to compensate the text so it will never get cut off.

Reply:
ohh okay, I see. I just tried it out. There comes a point when the height doesn't get taller, when the width is smaller than a singular world, but that should never happen. I'll get rid of it